python/mxnet/io.py

```diff
@@ -6,6 +6,7 @@
 import ctypes
 import logging
 import threading
+import h5py
```

Comment:
I changed it. On a side-note, how to specify h5py as a dependency for the tests?